docs/source/markdown/podman-login.1.md.in

```diff
@@ -71,7 +71,7 @@ print detailed information about credential store
 
 Add login credentials for specified registry to the default authorization file.
```

Comment:
Sorry, there seem to be more cases of s/authorization/authentiation/ in these man pages.